run.py

```diff
@@ -150,10 +190,16 @@ def before_request():
 
 @app.teardown_appcontext
 def teardown(exception):
+    '''
```

Comment:
This is incorrectly worded, and is just the logic of this method translated into "plain" English. Take a look at what the Jinja documentation has to say about teardown, and adapt your comment to that.